src/nvim/eval.c

```diff
@@ -16716,13 +16788,19 @@ static void f_timer_stop(typval_T *argvars, typval_T *rettv, FunPtr fptr)
     timer_stop(timer);
 }
 
+static void f_timer_stopall(typval_T *argvars, typval_T *unused, FunPtr fptr)
+{
+  timer_teardown();
```

Comment:
nitpick: I think this function then should change name, `teardown` implies that it will only will be called at exit.